crypto/ocsp/ocsp_integration_test.cc

```diff
@@ -18,6 +19,35 @@
 
 std::string GetTestData(const char *path);
 
+// Certificates always expire and that applies to the root certificates we're
+// using in our trust store too.
+static void isCertificateExpiring(X509 *cert) {
+  // Get the current time and add 1 year to it.
+  int64_t warning_time = time(nullptr) + 31536000;
+
+  if (X509_cmp_time_posix(X509_get_notAfter(cert), warning_time) < 0) {
+    fprintf(stdout, "\n\n\n\n");
+    X509_NAME_print_ex_fp(stdout, X509_get_subject_name(cert), 0, 0);
+    fprintf(stdout, " WILL EXPIRE IN A YEAR, PLEASE REPLACE ME WITH THE NEW"
+            " EXPECTED ROOT CERTIFICATE FROM THE ENDPOINT.\n\n\n\n");
+  }
+}
+
+static X509_STORE *SetupTrustStore() {
+  X509_STORE *trust_store = X509_STORE_new();
+  for (int i = 1; i <= 3; i++) {
+    bssl::UniquePtr<X509> ca_cert(CertFromPEM(
+        GetTestData(
+            std::string("crypto/ocsp/test/integration-tests/AmazonRootCA" +
+                        std::to_string(i) + ".pem")
+                .c_str())
+            .c_str()));
```

Comment:
Can you remove the outermost round trip through the std::string?
```suggestion
            ("crypto/ocsp/test/integration-tests/AmazonRootCA" +
                        std::to_string(i) + ".pem") .c_str());
```

Reply:
Both functions involved here take a `const *char`, but C++ doesn't allow directly appending of a value to a const char, thus the round trip :( 
I've used `ostringstream` to clean things up and make things prettier though